### PR TITLE
docs(webapps): document batch operation instance selection

### PR DESCRIPTION
**Pull request by tasso94**
related to CAM-14902

### CODE REVIEW

File: content/webapps/cockpit/batch/batch-operation.md [hunk 1]
```diff
@@ -35,8 +35,15 @@ It is possible to execute the following batch operations:
 - Correlate Message.
 
 After selecting the operation, fields may appear with additional data that is either optional or required to perform the operation. 
-When canceling running process instances, you can optionally select to skip custom listeners and provide a cancellation reason. In addition to the
-ability to specify the desired operation, a search query can be specified which defines the affected process instances.
+When canceling running process instances, you can optionally select to skip custom listeners and provide a cancellation reason. 
+
+Next, you can define the instances affected by the batch operation.
+Initially, all instances are displayed. You can use the filter bar above the list of instances to filter the displayed instances.
+You can select specific instances or click on the "Query" radio button to select all instances matching the filter.
```
Comment (danielkelemen) [inline content/webapps/cockpit/batch/batch-operation.md:42]: I think we use bold text when referring to elements the user has to click on.

```suggestion
You can select specific instances or click on the *Query* radio button to select all instances matching the filter.
```

File: content/webapps/cockpit/bpmn/process-instance-migration.md [hunk 2]
```diff
@@ -105,7 +105,15 @@ You can still edit the variables you have just added. For instance, if a variabl
...
+After finishing the **Set Variables** step, click the `Select Instances` button to proceed to instance selection. 
+Initially, all instances of the source process definition version are displayed. You can use the filter bar above the list of instances to filter the displayed instances.
+You can select specific instances or click on the "Query" radio button to select all instances matching the filter.
```
Comment (danielkelemen) [inline content/webapps/cockpit/bpmn/process-instance-migration.md:110]:
```suggestion
You can select specific instances or click on the *Query* radio button to select all instances matching the filter.
```

File: content/webapps/cockpit/bpmn/process-instance-modification.md [hunk 3]
```diff
@@ -85,8 +85,15 @@ The batch modification operation is handled similarly to the modification of a s
 
 {{< img src="../../img/cockpit-batch-modification-instance-selection.png" title="Instance Selection" >}}
 
+However, to get to the *Apply Modifications* modal first you have to click *Select Instances* which opens another modal which allows you to select the process instances the modification should be applied to.
+Initially, all instances are displayed. You can use the filter bar above the list of instances to filter the displayed instances.
+You can select specific instances or click on the "Query" radio button to select all instances matching the filter.
```
Comment (danielkelemen) [inline content/webapps/cockpit/bpmn/process-instance-modification.md:90]:
```suggestion
You can select specific instances or click on the *Query* radio button to select all instances matching the filter.
```

File: content/webapps/cockpit/bpmn/process-instance-restart.md [hunk 4]
```diff
@@ -45,7 +45,15 @@ When restarting a process instance, you have the option to *start before* or *st
 
 {{< img src="../../img/cockpit-restart-instance-selection.png" title="Instance Selection" >}}
 
-When you click on *Select Instances* a modal opens which allows you to select a single or a batch of process instances which should be restarted. Note, in order to find the relevant instances, you can apply filters. When you have made your selection you can proceed by clicking the red button. This leads you to the *Confirm Restart* modal.  
+When you click on *Select Instances*, a modal opens. Initially, all instances are displayed. 
+You can use the filter bar above the list of instances to filter the displayed instances.
+You can select specific instances or click on the "Query" radio button to select all instances matching the filter.
```
Comment (danielkelemen) [inline content/webapps/cockpit/bpmn/process-instance-restart.md:50]:
```suggestion
You can select specific instances or click on the *Query* radio button to select all instances matching the filter.
```

**Review by danielkelemen — CHANGES_REQUESTED** [review]
Looks good! I added comments about the quoting of ui elements.

File: content/webapps/cockpit/batch/batch-operation.md (hunk 1 above)
Comment (tasso94) [reply content/webapps/cockpit/batch/batch-operation.md]: Nice catch. Just a heads-up that single asterisks make it italics, and double asterisks make it bold.

**Review by christinaausley — APPROVED** [review]
Very minor grammatical adjustments in latest commit. LGTM 👍

**Pull request merged**